measurements/gunicorn_config.py

```diff
@@ -0,0 +1,9 @@
+from .config import PROMETHEUS_PORT
```

Comment:
I'd probably mention that this is the vanilla usage according to the docs

Reply:
What do you mean by this?

Reply:
That this code is exactly the default way of deploying if you read the docs.